components/client/upload-button.tsx

```diff
@@ -1,9 +1,12 @@
 "use client";
 
 import Button from "@mui/material/Button";
-import type { ReactElement } from "react";
+import type { ChangeEvent, ReactElement } from "react";
+import { useRef, useEffect, useState } from "react";
 import CloudUploadIcon from "@mui/icons-material/CloudUpload";
 import { styled } from "@mui/material/styles";
+import Snackbar from "@mui/material/Snackbar";
```

Comment:
@MadsHeilmann I'm a bit sceptical about having localized error handling. It would make much more sense to have a global handler that you can pass messages to

Reply:
I agree. Will open an issue on that.

Reply:
Rather, i have just fixed it.